detect_secrets/plugins/aws.py

```diff
@@ -15,3 +23,162 @@ class AWSKeyDetector(RegexBasedDetector):
     denylist = (
         re.compile(r'AKIA[0-9A-Z]{16}'),
```

Comment:
I wonder what the verify code would look like if we supported all the key types in https://github.com/dxa4481/truffleHogRegexes/pull/13/files

Reply:
We probably can cross that bridge, when we support them.

OR, if you have another valid prefixed key to test, that's easily testable too.

Reply:
++ for that 🤔 , I'll make an issue later for the other types 👍